### PR TITLE
JSS to Emotion migration, tabs

### PR DESCRIPTION
**Pull request by EstherDarkish**
Batch 9, focused on **tabs** in creation and modification dialogs.

### CODE REVIEW

File: src/components/dialogs/network-modifications/two-windings-transformer/creation/two-windings-transformer-creation-dialog-tabs.js [hunk 1]
```diff
@@ -44,22 +40,20 @@ const TwoWindingsTransformerCreationDialogTabs = ({
         name: `${PHASE_TAP_CHANGER}.${ENABLED}`,
     });
 
-    const getTabIndicatorClass = useCallback(
+    const getTabIndicatorStyle = useCallback(
```
Comment (sBouzols) [inline src/components/dialogs/network-modifications/two-windings-transformer/creation/two-windings-transformer-creation-dialog-tabs.js:43]: a bit of duplicated code, maybe we could improve this.

File: src/components/dialogs/network-modifications/two-windings-transformer/modification/two-windings-transformer-modification-dialog-tabs.js [hunk 2]
```diff
@@ -18,46 +16,42 @@ import {
...
-
     const phaseTapChangerEnabledWatch = useWatch({
         name: `${PHASE_TAP_CHANGER}.${ENABLED}`,
     });
     const ratioTapChangerEnabledWatch = useWatch({
         name: `${RATIO_TAP_CHANGER}.${ENABLED}`,
     });
 
-    const getTabIndicatorClass = useCallback(
+    const getTabIndicatorStyle = useCallback(
```
Comment (sBouzols) [inline src/components/dialogs/network-modifications/two-windings-transformer/modification/two-windings-transformer-modification-dialog-tabs.js:41]: same

**Review by sBouzols — APPROVED** [review]
approved
But we could move `getTabIndicatorStyle` and `getTabStyle` as function which is shared between the three components

File: src/components/dialogs/network-modifications/two-windings-transformer/creation/two-windings-transformer-creation-dialog-tabs.js (hunk 1 above)
Comment (sBouzols) [reply src/components/dialogs/network-modifications/two-windings-transformer/creation/two-windings-transformer-creation-dialog-tabs.js]: thanks

File: src/components/dialogs/network-modifications/two-windings-transformer/modification/two-windings-transformer-modification-dialog-tabs.js (hunk 2 above)
Comment (sBouzols) [reply src/components/dialogs/network-modifications/two-windings-transformer/modification/two-windings-transformer-modification-dialog-tabs.js]: thanks

**Review by sBouzols — APPROVED** [review]

**Review by souissimai — APPROVED** [review]

**Pull request merged**